src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw

```diff
@@ -661,4 +661,16 @@
   <data name="CloseOtherPanesCommandKey" xml:space="preserve">
     <value>Close all other panes</value>
   </data>
+  <data name="SelectOutputNextCommandKey" xml:space="preserve">
+    <value>Select next command output</value>
+  </data>
+  <data name="SelectOutputPreviousCommandKey" xml:space="preserve">
+    <value>Select previous command output</value>
+  </data>
+  <data name="SelectCommandNextCommandKey" xml:space="preserve">
+    <value>Select next command</value>
+  </data>
+  <data name="SelectCommandPreviousCommandKey" xml:space="preserve">
+    <value>Select previous command</value>
+  </data>

```

Comment:
```suggestion
  <data name="SelectCommandNextCommandKey" xml:space="preserve">
    <value>Select next prompt</value>
  </data>
  <data name="SelectCommandPreviousCommandKey" xml:space="preserve">
    <value>Select previous prompt</value>
  </data>
```

Would that be more clear? Should we rename the action to `selectPrompt`?

Reply:
I'm gonna hard NAK this - the "prompt" is the `C:\Users\migrie>` part. The "command" is `ping 8.8.8.8`, and the output is {you get it}.

I didn't think there was value in just selecting the prompts, but commands and output made sense.